hack/test-go.sh

```diff
@@ -113,7 +129,13 @@ if [[ -n "${KUBE_COVER}" && -n "${OUTPUT_COVERAGE}" ]]; then
 
   rm -rf $OUTPUT_COVERAGE/$OS_GO_PACKAGE
 else
-  nice go test $KUBE_RACE $KUBE_TIMEOUT $KUBE_COVER "${@:2}" $test_packages
+  nice go test $verbose $KUBE_RACE $KUBE_TIMEOUT $KUBE_COVER "${@:2}" $test_packages | tee "${JUNIT_OUTPUT_FILE}"
```

Comment:
`grep` the output that's displayed on the screen to keep the stdout looking like the non verbose `go test` output.  The verbose output is too busy to read.